adding rum-cli-examples (#8984)
aws/aws-cli: awscli/examples/rum/update-rum-metric-definition.rst

> **To modifies one existing metric definition for CloudWatch RUM extended metric**

Reviewer: ```suggestion
**To modify an existing metric definition for CloudWatch RUM extended metric**
```

Author: updated

> The following ``update-rum-metric-definition`` modifies one existing metric definition for CloudWatch RUM extended metrics. If the command succeeds, no output is returned. ::

Reviewer: ```suggestion
The following ``update-rum-metric-definition`` example modifies one existing metric definition for CloudWatch RUM extended metrics. ::
```

Author: updated

> 	aws rum update-rum-metric-definition \

Reviewer: Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general

		--app-monitor-name MyWebApp \
		--destination CloudWatch \
		--metric-definition 'Namespace=PageVisit,Name=AmazonVisitCount,DimensionKeys={event_details.current_url=URL},EventPattern="{\"metadata\":{\"browserName\":[\"Chrome\"]},\"event_type\":[\"my_custom_event\"],\"event_details\":{\"current_url\":[\"amazonaws.com\"]}}"' \
		--metric-definition-id 8b67gh3-r678-q987-a576-b9fj7kf234kd

> 

Reviewer: ```suggestion

This command produces no output.

```

Author: updated
